Index: src/main/java/it/polimi/ingsw/model/Game.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package it.polimi.ingsw.model;\r\n\r\npublic class Game {\r\n    private Game instance;\r\n    private int maxPlayerNumber;\r\n    private boolean lastTurn;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Game.java b/src/main/java/it/polimi/ingsw/model/Game.java
--- a/src/main/java/it/polimi/ingsw/model/Game.java	(revision b4cdfd7aa42eaa5ac41a8a076622c7b3d74cca12)
+++ b/src/main/java/it/polimi/ingsw/model/Game.java	(date 1679306701150)
@@ -4,4 +4,5 @@
     private Game instance;
     private int maxPlayerNumber;
     private boolean lastTurn;
+    private List<Pla>
 }
Index: src/main/java/it/polimi/ingsw/model/Player.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Player.java b/src/main/java/it/polimi/ingsw/model/Player.java
new file mode 100644
--- /dev/null	(date 1679306545870)
+++ b/src/main/java/it/polimi/ingsw/model/Player.java	(date 1679306545870)
@@ -0,0 +1,15 @@
+package it.polimi.ingsw.model;
+
+import java.util.List;
+
+public class Player {
+    private String nickname;
+    private  boolean connected;
+    private int playerScore;
+    private boolean EndGameToken;
+    private Shelf shelf;
+    private List<ScoringToken> scoringToken;
+    private List<CommonTargetCard> commonTargetCard;
+    private PersonalTargetCard personalTargetCard;
+
+}
Index: src/main/java/it/polimi/ingsw/model/Token.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Token.java b/src/main/java/it/polimi/ingsw/model/Token.java
new file mode 100644
--- /dev/null	(date 1679306036308)
+++ b/src/main/java/it/polimi/ingsw/model/Token.java	(date 1679306036308)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class Token {
+}
Index: src/main/java/it/polimi/ingsw/model/EndGameToken.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/EndGameToken.java b/src/main/java/it/polimi/ingsw/model/EndGameToken.java
new file mode 100644
--- /dev/null	(date 1679306014533)
+++ b/src/main/java/it/polimi/ingsw/model/EndGameToken.java	(date 1679306014533)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class EndGameToken {
+}
Index: src/main/java/it/polimi/ingsw/model/CommonTargetCard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/CommonTargetCard.java b/src/main/java/it/polimi/ingsw/model/CommonTargetCard.java
new file mode 100644
--- /dev/null	(date 1679306123756)
+++ b/src/main/java/it/polimi/ingsw/model/CommonTargetCard.java	(date 1679306123756)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class CommonTargetCard {
+}
Index: src/main/java/it/polimi/ingsw/model/ScoringToken.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/ScoringToken.java b/src/main/java/it/polimi/ingsw/model/ScoringToken.java
new file mode 100644
--- /dev/null	(date 1679306074888)
+++ b/src/main/java/it/polimi/ingsw/model/ScoringToken.java	(date 1679306074888)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class ScoringToken {
+}
Index: src/main/java/it/polimi/ingsw/model/Target.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Target.java b/src/main/java/it/polimi/ingsw/model/Target.java
new file mode 100644
--- /dev/null	(date 1679306123743)
+++ b/src/main/java/it/polimi/ingsw/model/Target.java	(date 1679306123743)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class Target {
+}
Index: src/main/java/it/polimi/ingsw/model/PersonalTargetCard.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/PersonalTargetCard.java b/src/main/java/it/polimi/ingsw/model/PersonalTargetCard.java
new file mode 100644
--- /dev/null	(date 1679306123798)
+++ b/src/main/java/it/polimi/ingsw/model/PersonalTargetCard.java	(date 1679306123798)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class PersonalTargetCard {
+}
Index: src/main/java/it/polimi/ingsw/model/Shelf.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Shelf.java b/src/main/java/it/polimi/ingsw/model/Shelf.java
new file mode 100644
--- /dev/null	(date 1679306167270)
+++ b/src/main/java/it/polimi/ingsw/model/Shelf.java	(date 1679306167270)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class Shelf {
+}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"MavenProjectsManager\">\r\n    <option name=\"originalFiles\">\r\n      <list>\r\n        <option value=\"$PROJECT_DIR$/pom.xml\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_19\" default=\"true\" project-jdk-name=\"19\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision b4cdfd7aa42eaa5ac41a8a076622c7b3d74cca12)
+++ b/.idea/misc.xml	(date 1679004367874)
@@ -8,7 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="19" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" project-jdk-name="17" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
Index: src/main/java/it/polimi/ingsw/model/Item.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Item.java b/src/main/java/it/polimi/ingsw/model/Item.java
new file mode 100644
--- /dev/null	(date 1679306195945)
+++ b/src/main/java/it/polimi/ingsw/model/Item.java	(date 1679306195945)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class Item {
+}
Index: src/main/java/it/polimi/ingsw/model/Board.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/model/Board.java b/src/main/java/it/polimi/ingsw/model/Board.java
new file mode 100644
--- /dev/null	(date 1679306167282)
+++ b/src/main/java/it/polimi/ingsw/model/Board.java	(date 1679306167282)
@@ -0,0 +1,4 @@
+package it.polimi.ingsw.model;
+
+public class Board {
+}
